### DISCUSSION TITLE
This thing is so fantastic

### DISCUSSION THREAD

**Discussion by qpwo** [appy-one/acebase#51] [General]
I am in your debt, Ewout Stortenbeker. It is weird to me that one database would stand out among thousands of apparently similar options, but this thing really is better

**Comment by appy-one** [comment]
Thanks! A lot of time has gone (and still goes) into this project. 
Spread the word!

**Reply by Bessonov** [reply]
I think too that this project is underrated. Not ace user yet, but interested in it as an alternative to rxdb or watermelondb and before writing own thing. I didn't found any information about data migration. How to handle it?
